.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build theme for WP sites
 
 on: 
   push:
-    branches: [theme]
+    branches: [theme, development]
```

Comment:
Nie mamy takiego brancha w naszym repo, mógłbyś wycofać tę zmianę?

Reply:
Na dzień dzisiejszy jesteśmy w stanie wycofać. Zrobię to wraz z domyślnymi wartościami w palecie kolorów.